internal/turso/databases.go

```diff
@@ -25,20 +25,30 @@ type Database struct {
 	IsSchema      bool `json:"is_schema"`
 }
 
-type DatabasesClient client
-
-func (d *DatabasesClient) List(options map[string]string) ([]Database, error) {
-	path := d.URL("")
+type DatabaseListOptions struct {
+	Group  string `json:"group,omitempty"`
+	Schema string `json:"schema,omitempty"`
```

Comment:
⛏️: feel free to drop the annotations, since we're not encoding/decoding JSON.
```suggestion
	Group  string
	Schema string
```